lib/meta_tags/text_normalizer.rb

```diff
@@ -68,7 +68,7 @@ def self.safe_join(array, sep = $,)
     # space characters squashed into a single space.
     #
     def self.cleanup_string(string)
-      strip_tags(string).gsub(/\s+/, ' ').strip
+      strip_tags(string).gsub(/\s+/, ' ').gsub(/"/, "\'").strip
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.